widgets-ap/src/providers/ModeProvider.js

```diff
@@ -0,0 +1,20 @@
+import React, { createContext, useState } from "react";
+export const ModeContext = createContext();
+
+const ModeProvider = (props) => {
+    const [toggle, setToggle] = useState(sessionStorage.getItem("DarkMode") === "true");
```

Comment:
It would be cleaner to create a variable to inset this logic outside the `useState`